IPS/IPS-IOC-01App/src/cryomagnet.st

```diff
@@ -1,6 +1,6 @@
-program cryomagnet("FIELD,FIELD_SETPOINT,FIELD_SETPOINT_ALARM,FIELD_SETPOINT_RAW,FIELD_SETPOINT_READBACK,PERSISTENT,MAGNET_FIELD,HEATER_STATUS,HEATER_STATUS_SP,HEATER_WAIT_TIME,ACTIVITY,ACTIVITY_SP,SWEEPMODE,SWEEPMODE_SP,SUPPLYVOLTAGE_STABLE,SWEEP_ACTIVE,STATEMACHINE,HEATER_ONTIME_OK")
+program cryomagnet("FIELD,FIELD_SETPOINT,FIELD_SETPOINT_ALARM,FIELD_SETPOINT_RAW,FIELD_SETPOINT_READBACK,PERSISTENT,MAGNET_FIELD,HEATER_STATUS,HEATER_STATUS_SP,HEATER_WAIT_TIME,ACTIVITY,ACTIVITY_SP,SWEEPMODE,SWEEPMODE_SP,SUPPLYVOLTAGE_STABLE,SWEEP_ACTIVE,STATEMACHINE")
 
-#include "ibexSeqPVmacros.h"
+#include "seqPVmacros.h"
```

Comment:
I think want to keep using  ibexSeqPVmacros.h ?

Reply:
We seem to be reversing some of https://github.com/ISISComputingGroup/EPICS-ioc/pull/771 ?

Reply:
we do, those changes look later than 12.0.1 as well

Reply:
just one thought i had - @davidkeymer do you think you may have set the IPS ioc on your machine for testing? Perhaps the changes on WISH weren't actually the last ones used/tested?